bucket/lib.rs

```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
     }
 
     impl DdcBucket {
-        /// Create a new cluster and return its `cluster_id`.
-        ///
-        /// The caller will be its first manager.
+        /// Description:
+        ///     Create a new cluster.
         ///
-        /// The cluster is split in a number of vnodes. The vnodes are assigned to the given physical nodes in a round-robin. Only nodes of providers that trust the cluster manager can be used (see `node_trust_manager`). The assignment can be changed with the function `cluster_replace_node`.
+        /// Logic:
```

Comment:
Can we wrap it with some formatting here so section names will be more visible in final .md file?

Reply:
updated